pkg/config/set.go

```diff
@@ -0,0 +1,48 @@
+package config
+
+// FileReference is a named file path.
+type FileReference struct {
+	Name string `json:"name"`
+	Path string `json:"path"`
+}
+
+// FileSet is a set of FileReference indexed by name.
+type FileSet []FileReference
```

Comment:
Just curious, can you use `map` in this module?

Reply:
No. It is by design to keep the order. Note that `map` in golang does not persve order or sorted.